tests/test_grid.py

```diff
@@ -0,0 +1,141 @@
+"""Test module for grid.py.
+
+NAME
+test_grid.py
+
+DESCRIPTION
+This module tests the following functions from grid.py:
+
+- class CoreGridConfig:
+    Configure the `core.grid` module.
+    This data class is used to setup the arrangement of grid cells to be used in
+    running a `virtual_rainforest` simulation.
+
+- CoreGrid._make_square_grid(cell_area: float, cell_nx: int, cell_ny: int) -> dict:
+    Create a square grid.
+
+- CoreGrid._make_hex_grid(cell_area: float, cell_nx: int, cell_ny: int) -> dict:
+    Create a hexagonal grid.
+
+- TODO CoreGrid.export geojson
+    Export grid system as geojson file.
+    outfile: A self.cell_dict returned by a make_square_grid or make_hex_grid call.
+
+- TODO Once it is extracted from where it is, _feature.
+
+"""
+
+import pytest
+from hypothesis import given
+from hypothesis.strategies import integers
+
+
+def test_CoreGridConfig():
+    """Test if CoreGridConfig().
+
+    Check if function created an object with default values for the following
+    properties: grid_type, cell_contiguity, cell_area, cell_nx, cell_ny.
+    """
+    from virtual_rainforest.core.grid import CoreGridConfig
+
+    obj_1 = CoreGridConfig()
+    assert obj_1.grid_type == "square"
+    assert obj_1.cell_contiguity == "rook"
+    assert obj_1.cell_area == 100
+    assert obj_1.cell_nx == 10
+    assert obj_1.cell_ny == 10
+
+
+@given(integers(min_value=0, max_value=9), integers(min_value=0, max_value=9))
```

Comment:
Tests in MacOS are failing for some uinknown reason - I can reproduce this in my laptop. To solve this, remove the deadline limitation for this test:

```python
from hypothesis import given, settings
...
@settings(deadline=None)
@given(integers(min_value=0, max_value=9), integers(min_value=0, max_value=9))
def test_make_square_grid(x, y):
    ...
```

Reply:
It's the same for me, it passes all the tests on my laptop. I will try your suggestions

Reply:
all tests pass now.